### DISCUSSION TITLE
adding users without email

### DISCUSSION THREAD

**Discussion by sysadminpower2019** [dani-garcia/vaultwarden#2471] [General]
hello,

ive managed to set up vaultwarden after a lot of struggling. Now i've come to the issue of needing to set up an smtp server to send a password. I tried to set up an email server using postfix and it seems to be partially working none the less vaultwarden doesnt seem to connect to it.

my question is how can i manage users without having to send out emails. When i try to use the create account button it creates a user in a totally different vault / organization and there is no way that i can find to connect it to the main organization that i created for the 'first account'

is there a way to manager users directly without having to relay on email? i tried to use the bitwarden CLI interface and it also isnt extremely clear if there is a way to do this.

**Comment by BlackDex** [comment]
I suggest to read: https://bitwarden.com/help/getting-started-organizations/ on how Organizations work in Bitwarden.

If a user already exists in the same self-hosted environment and you can just invite that user to any org you want, if the user doesn't exists a user will be created, this is for both Bitwarden and Vaultwarden.

Managing Organizations is only possible via the web-vault, not any other client.

Further regarding some Vaultwarden specific items.
- You do not need SMTP configured
- You still need to use a mail address as a login, but it may be an invalid address (Though i suggest to use a valid).
- Without SMTP it will accept invites to organizations directly after the invite.
- Users created the via Admin Interface can simply be invited to any origanization on that same self-hosted instance.

As a remark. You can't link users from different self-hosted environments. Or the Bitwarden provided cloud environment.
All users have to be on exact same instance.

**Comment by af3556** [comment]
https://github.com/dani-garcia/vaultwarden/discussions/4531 describes how to add users without using email.
